modules/imgproc/test/test_color.cpp

```diff
@@ -1077,7 +1068,7 @@ double CV_ColorLabTest::get_success_error_level( int /*test_case_idx*/, int i, i
 {
     int depth = test_mat[i][j].depth();
     // j == 0 is for forward code, j == 1 is for inverse code
-    return (depth ==  CV_8U) ? (srgb ? 32 : 8) :
+    return (depth ==  CV_8U) ? (srgb ? 37 : 8) :
```

Comment:
Why is it changed?

Reply:
I came to conclusion that these thresholds have been chosen experimentally and after changing RNG seeds some tests can fail. We've observed the same problem with these tests in another PR: https://github.com/opencv/opencv/pull/16189#discussion_r996149383

I specifically moved this fix to separate commit, so it can be backported to 4.x later.